docker/logging.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Feel free to come up with a simpler log config. I went down the rabbithole with this.

Reply:
I adjusted it so that it doesn't log to files inside the container.